protocols/kad/Cargo.toml

```diff
@@ -21,7 +21,7 @@ log = "0.4"
 libp2p-core = { version = "0.36.0", path = "../../core", default-features = false }
 libp2p-swarm = { version = "0.39.0", path = "../../swarm" }
 prost = "0.11"
-rand = "0.7.2"
+rand = "0.8"
```

Comment:
Pull request needs to bump `libp2p-kad` to `v0.40.1`. See comment on `libp2p-mplex` above.